lib/utils.nix

```diff
@@ -11,69 +12,78 @@ with lib;
     "__empty" = null;
   };
 
-  /**
-    Turn all the keys of an attrs into raw lua.
+  /*
```

Comment:
Alejandra doesn't support [RFC145](https://github.com/nixos/rfcs/blob/master/rfcs/0145-doc-strings.md#migration-example) docstrings!

Reply:
There is a [patch](https://github.com/kamadorueda/alejandra/pull/401) for this issue, though we probably want to avoid compiling a rust program each time we format our code